tests/fs_test.py

```diff
@@ -11,6 +11,18 @@
 
 from gi.repository import BlockDev, GLib
 
+fs_spec = BlockDev.PluginSpec()
+fs_spec.name = BlockDev.Plugin.FS
+BlockDev.ensure_init(require_plugins=[fs_spec])
+try:
+    ntfs_avail = BlockDev.fs_is_tech_avail(BlockDev.FSTech.NTFS, BlockDev.FSTechMode.MKFS) and \
+                 BlockDev.fs_is_tech_avail(BlockDev.FSTech.NTFS, BlockDev.FSTechMode.RESIZE) and \
+                 BlockDev.fs_is_tech_avail(BlockDev.FSTech.NTFS, BlockDev.FSTechMode.REPAIR) and \
+                 BlockDev.fs_is_tech_avail(BlockDev.FSTech.NTFS, BlockDev.FSTechMode.CHECK) and \
+                 BlockDev.fs_is_tech_avail(BlockDev.FSTech.NTFS, BlockDev.FSTechMode.SET_LABEL)
```

Comment:
The modes can be combined with ``|`` to create a bitmask. So just a single call of the ``fs_is_tech_avail()`` function is needed here.